src/log4net/log4net.csproj

```diff
@@ -63,10 +62,10 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)'=='Debug' ">
     <DefineConstants>TRACE;DEBUG;$(DefineConstants)</DefineConstants>
+    <GeneratePackageOnBuild>$(GeneratePackages)</GeneratePackageOnBuild>
```

Comment:
why auto-generate packages just for debug builds? this will slow down any TDD effort, btw, as you'd be building packages at every point in the red-green-refactor cycle, but I'm assuming there's a good reason?

Reply:
Please see line 69.
For Debug-Builds you can turn on package generation via "GeneratePackages" (default: false).
For Release-Builds they are always on (assuming you do TDD with Debug binaries ;-)